src/assets/footer/spotify-icon.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="31" height="31" viewBox="0 0 31 31" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think its better to use MUI icon instead.

Reply:
Spotify doesn't have MUI icon. I'll change the width and height to percentage

Reply:
Btw responsiveness fixed for tablet view.